Polyhedron/demo/Polyhedron/Plugins/Camera_position/CMakeLists.txt

```diff
@@ -7,5 +7,6 @@ polyhedron_demo_plugin(camera_positions_plugin
     ${cameraUI_FILES}
     KEYWORDS Viewer)
 
+add_dependencies(camera_positions_plugin demo_framework)
 target_link_libraries( camera_positions_plugin PUBLIC demo_framework)
 target_link_libraries(camera_positions_plugin PUBLIC demo_framework)
```

Comment:
Linking a plugin to a target is unsufficient to be sure it will not be built before (or at the same time as) the dependency, It was one of the few fixes I put into place before encountering the `.tlog` bug. I figured it could not harm to leave them here.